src/main/java/com/mg105/entities/Item.java

```diff
@@ -0,0 +1,25 @@
+package com.mg105.entities;
+
+
+public abstract class Item {
+    final String name;
+    final String description;
```

Comment:
Should these be private and not default scope? If they're final anyways why not make them public?

Reply:
Yeah I agree the only thing is that I remember them saying to back things public and just write getters (but I'm not sure if that extends to constants)